ci/config/molecule.yaml

```diff
@@ -98,3 +98,8 @@
 - job:
     name: cifmw-molecule-shiftstack
     nodeset: centos-9-crc-2-48-0-xl-ibm
+- job:
+    name: cifmw-molecule-tofu
+    nodeset: centos-9-crc-2-48-0-xl
```

Comment:
nit: I don't think setting nodeset is needed here.

Reply:
I'm not sure about that. I just left what was initially done [here](https://github.com/openstack-k8s-operators/ci-framework/pull/2909/files#diff-897128cff4fe0d5b2c8aba92a2c8b3ed825cc44ba495b87a1ebe5353d0e42a7fL9)